incubator/hnc/internal/stats/metrics.go

```diff
@@ -107,6 +108,13 @@ var (
 		Aggregation: ocview.LastValue(),
 		TagKeys:     []tag.Key{KeyNamespaceCondition, KeyNamespaceCritical},
 	}
+
+	objectOverwritesTotalView = &ocview.View{
+		Name:        "hnc/reconcilers/object/object_overwrites_total",
```

Comment:
Maybe let's call this `overwrites_total`, since it's already nested under `object`. I know that this is to line up with `object_writes_total` on line 97 but we should probably change that too (feel free to change it if you like in this PR).

Reply:
done